packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Header/index.tsx

```diff
@@ -63,74 +77,108 @@ const NftHeader: React.FC<InjectedFormProps<{}, Props> & Props> = ({
 
   return (
     <Wrapper>
-      <TabsContainer>
-        <TabMenu>
-          <TabMenuItem onClick={() => setActiveTab('explore')} selected={activeTab === 'explore'}>
-            Explore
-          </TabMenuItem>
-          <TabMenuItem
-            onClick={() => setActiveTab('my-collection')}
-            selected={activeTab === 'my-collection'}
+      {seaStatus !== '' && seaStatus !== 'All Systems Operational' && (
+        <ItemWrapper>
```

Comment:
Consider moving this to it's own component

<OpenseaStatus />

Inside that component do the call to fetch the status in the `useEffect` hook.

Reply:
You've already correctly set up the `fetchOpenseaStatus` saga, just need to call that from the new component's `useEffect` hook